sessionAction.js

```diff
@@ -0,0 +1,45 @@
+function singInUser(email, password) {
+  inizializeRequest(
+    singInUrl,
+    {email: email, password: password},
+    'POST',
+    function(response) {
+      localStorage.setItem('email', email);
+      localStorage.setItem('auth_token', response);
+    },
+    function(response) {
+      alert('blad logowania');
```

Comment:
we need to be prepared for error statuses: 404, 403 and 500, and use bootstrap alert popups here. can be done in another PR tho